client/src/pages/Profile.js

```diff
@@ -1,5 +1,34 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import { ReactComponent as User } from '../assets/user.svg';
+import { Loader } from '../components';
 
-const Profile = () => <h1> Profile </h1>;
+const Profile = () => {
+  const [profile, setProfile] = useState();
+
+  const getProfile = async () => {
+    const { data } = await axios.get('api/v1/profile');
```

Comment:
You can destruct this line to:
```suggestion
    const {
        data: [userData],
      } = await axios.get('api/v1/profile');
```